.github/workflows/deploy-cloudrun.yaml

```
@@ -0,0 +1,61 @@
+        with:
+          image: ${{ inputs.registry }}${{ inputs.artifact-path }}:${{ inputs.artifact-tag }}
+          region: ${{ inputs.region }}
+          job: ${{ inputs.job-name }}
+      - name: 'deploy cloudrun service'
+        if: inputs.service-name != ''
+        uses: 'google-github-actions/deploy-cloudrun@v3'
+        with:
+          image: ${{ inputs.registry }}${{ inputs.artifact-path }}:${{ inputs.artifact-tag }}
+          region: ${{ inputs.region }}
+          service: ${{ inputs.service-name }}
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)